README.md

```diff
@@ -9,7 +9,7 @@ The integration with Amazon S3 relies on [AWS Lambda](https://docs.aws.amazon.co
  * Code entry type: `Upload a .ZIP file`
  * Upload our LogDNA Lambda function [.ZIP File](https://github.com/logdna/logdna-s3/releases/latest/download/logdna-s3.zip).
  * Handler: `index.handler`
- * Runtime: `Node.js.16.x`
+ * Runtime: `Node.js.20.x`
```

Comment:
I would suggest we call this the "Minimum Runtime Version", since the question has been asked a few times about support for newer versions.